src/components/main/TableToolbar.js

```diff
@@ -57,6 +58,8 @@ const TableToolbar = (props) => {
         </Typography>
       )}
 
+      {numSelected > 0 ? null : itemSearchInput}
```

Comment:
Would the following work? `!numSelected  &&  itemSearchInput`
Perhaps you can even merge with the following condition.